src/js/eva.js

```diff
@@ -473,9 +474,9 @@ Eva.prototype = {
 
         var twitterWidgetEl = document.getElementById('twitter-widget');
         twitterWidgetEl.innerHTML = "";
-        twitterWidgetEl.innerHTML = '<a  class="twitter-timeline" height=100 href="https://twitter.com/EBIvariation"  data-widget-id="437894469380100096">Tweets by @EBIvariation</a>';
+        twitterWidgetEl.innerHTML = '<a  class="twitter-timeline"  href="https://twitter.com/EBIvariation"  height="100" data-widget-id="437894469380100096" data-chrome="noheader nofooter noborders transparent">Tweets by @EBIvariation</a>';
         $.getScript('//platform.twitter.com/widgets.js', function () {
```

Comment:
Can this be removed or would it prevent the Twitter widget from being displayed?